web/app/features/Bookmarks/components/Bookmarks/Bookmarks.tsx

```diff
@@ -26,8 +24,6 @@ export function Bookmarks({ bookmarks }: BookmarksProps) {
       <>
         <Headings.Content>Bookmarks</Headings.Content>
         {bookmarks.map(bookmark => {
-          const disabled = useDisabled('bookmarkId', bookmark.id);
```

Comment:
**dev note**: The issue here was that we were executing the hook inside a loop and according to the rules of hooks, they can only be called in the top level of our component's function.